runtime/compiler/z/codegen/J9BCDTreeEvaluator.cpp

```diff
@@ -5138,6 +5138,115 @@ J9::Z::TreeEvaluator::pdchkEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+    *                                     010 (4e,60) for sign separate (leading/trailing).
+    *                                     000 all sign codes are considered valid (hex) (A-F).
+    *    - Digits Count (DC)            : Integer specifying number of bytes to be verified. Does not include sign byte.
+    */
+   uint16_t zonedDecimalInfo = 0x0; // I3 operandgit d
+   uint8_t stc = 0x0;
+
+   int8_t dsc = static_cast<int8_t>(precision - bytesWithSpacesConst);
+
+   if (dataType == TR::ZonedDecimalSignTrailingSeparate) // Sign trailing separate
+      {
+      zonedDecimalInfo = 0x2; // set SSC bit
+      stc = 0x2;
+      dsc = 0x0;
+      }
+   else if (dataType == TR::ZonedDecimalSignLeadingEmbedded) // Sign leading embedded
+      {
+      zonedDecimalInfo = 0x1; // set LC bit
+      }
+   else if (dataType == TR::ZonedDecimalSignLeadingSeparate) // Sign leading separate
+      {
+      zonedDecimalInfo = 0x3; // set SSC and LS bits
+      stc = 0x2;
+      dsc = 0x0;
+      }
+   zonedDecimalInfo = (zonedDecimalInfo << 5) | dsc;
+   zonedDecimalInfo = (zonedDecimalInfo << 3) | stc;
+   zonedDecimalInfo = (zonedDecimalInfo << 5) | precision; // DC
+
+   // Must use decimalLength because precision does not equal length when sign is separate
+   int8_t firstByteIndexToLoad = decimalLength - 1;
+   TR::MemoryReference *zonedDecimalLowMR = generateS390MemoryReference(*sourceMR, 0, cg);
+   TR::MemoryReference *zonedDecimalHighMR = NULL;
+   if (decimalLength > TR_VECTOR_REGISTER_SIZE)
+      {
+      zonedDecimalHighMR = zonedDecimalLowMR;
+      firstByteIndexToLoad = firstByteIndexToLoad - TR_VECTOR_REGISTER_SIZE;
+      generateVSIInstruction(cg, TR::InstOpCode::VLRL, node, vZondedHighReg, zonedDecimalHighMR, firstByteIndexToLoad);
+
+      zonedDecimalLowMR = generateS390MemoryReference(*sourceMR, decimalLength - TR_VECTOR_REGISTER_SIZE, cg);
+      firstByteIndexToLoad = TR_VECTOR_REGISTER_SIZE - 1;
+      }
+
+   generateVSIInstruction(cg, TR::InstOpCode::VLRL, node, vZondedLowReg, zonedDecimalLowMR, firstByteIndexToLoad);
+   generateVRIlInstruction(cg, TR::InstOpCode::VTZ, node, vZondedHighReg, vZondedLowReg, zonedDecimalInfo);
+   generateRRInstruction(cg, TR::InstOpCode::IPM, node, chkResultReg, chkResultReg);
+   if(cg->comp()->target().is64Bit())
+      {
+      generateRRInstruction(cg, TR::InstOpCode::LLGTR, node, chkResultReg, chkResultReg);
+      generateRSInstruction(cg, TR::InstOpCode::SRLG, node, chkResultReg, chkResultReg, 28);
+      }
+   else
+      {
+      generateRSInstruction(cg, TR::InstOpCode::SRL, node, chkResultReg, 28);
+      }
+
+   if (vZondedLowReg) cg->stopUsingRegister(vZondedLowReg);
+   if (vZondedHighReg) cg->stopUsingRegister(vZondedHighReg);
+
+   node->setRegister(chkResultReg);
+   cg->decReferenceCount(child);
+   cg->decReferenceCount(node->getSecondChild());
+   cg->traceBCDExit("zdchk",node);
```

Comment:
Nit. Space after the comma please.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/20507/commits/01b9c78830f0f358730a6f011415c9637cdc6f33